Dockerfile

```diff
@@ -4,8 +4,10 @@ EXPOSE 8080
 
 ADD sendEmail-epel-7.repo /etc/yum.repos.d/
 
+ENV NAGIOS_PLUGINS /opt/rhmap/nagios/plugins/
```

Comment:
Or really, if we don't need this env in the runtime, I'd just replace it's usages below.

We don't give env vars to things like `/var/log/nagios/spool/checkresults`, and other long paths, so having one for nagios plugins is at least inconsistent.